packages/fxa-settings/src/pages/ResetPassword/ResetPassword/index.tsx

```diff
@@ -83,21 +82,20 @@ const ResetPassword = ({
 
   return (
     <AppLayout>
-      <CardHeader
-        headingText="Reset your password"
-        headingTextFtlId="password-reset-start-heading"
-      />
+      <FtlMsg id="password-reset-start-heading">
+        <h1 className="card-header text-start">Reset your password</h1>
+      </FtlMsg>
 
       {errorMessage && (
         <Banner type={BannerType.error}>
           <p>{errorMessage}</p>
         </Banner>
       )}
 
-      <FtlMsg id="password-reset-body">
-        <p className="my-6">
-          Enter your email and we’ll send you a confirmation code to confirm
-          it’s really you.
+      <FtlMsg id="password-reset-body-2">
+        <p className="text-start mt-2 mb-6">
```

Comment:
Instead of setting individual elements to `text-start`, I believe you can just do `<AppLayout cardClass="card-base">`.

(We could switch all of these at once to left-align but would need some review I think...)

Reply:
Looks like this also impacts our banners, so I might hold off until we've updated the styling there to work with `text-start`.